app/src/main/java/com/ortin/gesturetranslator/presentation/screens/SettingsScreen.kt

```diff
@@ -0,0 +1,39 @@
+package com.ortin.gesturetranslator.presentation.screens
+
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.ortin.gesturetranslator.presentation.ui.theme.SurfaceText
+import com.ortin.gesturetranslator.presentation.ui.theme.ThemedButton
+import com.ortin.gesturetranslator.presentation.ui.theme.OrtinTheme
+import com.ortin.gesturetranslator.presentation.ui.theme.loginPlease
+
+@Composable
+fun SettingsScreen() {
+    Column(
+        modifier = Modifier.padding(20.dp, 0.dp)
```

Comment:
```suggestion
        modifier = Modifier.padding(horizontal = 20.dp)
```

Если я тут не прав - то поправь. Всегда пиши в Compose что конкретно ты делаешь. Если это отступ - то какой и где) Задавай параметры